include/af/arith.h

```diff
@@ -322,79 +327,87 @@ namespace af
+    /// \param[in] in input array
+    /// \return       complex array
     AFAPI array complex(const array& in);
  
     /// C++ Interface to create a complex array from two real arrays.
     ///
-    /// \param[in] real_ a real array to be assigned as the real component of the returned complex array
-    /// \param[in] imag_ a real array to be assigned as the imaginary component of the returned complex array
-    /// \return the returned complex array
+    /// \param[in] real_ input array to be assigned as the real component of
+    ///                  the returned complex array
+    /// \param[in] imag_ input array to be assigned as the imaginary component
+    ///                  of the returned complex array
+    /// \return          complex array
     AFAPI array complex(const array &real_, const array &imag_);
 
-    /// C++ Interface to create a complex array from a single real array for the real component and a single scalar for each imaginary component.
+    /// C++ Interface to create a complex array from a single real array for
+    /// the real component and a single scalar for each imaginary component.
     ///
-    /// \param[in] real_ a real array to be assigned as the real component of the returned complex array
-    /// \param[in] imag_ a single scalar to be assigned as the imaginary component of each value of the returned complex array
-    /// \return the returned complex array
+    /// \param[in] real_ input array to be assigned as the real component of
+    ///                  the returned complex array
+    /// \param[in] imag_ single scalar to be assigned as the imaginary
+    ///                  component of each value of the returned complex array
+    /// \return          complex array
     AFAPI array complex(const array &real_, const double imag_);
 
-    /// C++ Interface to create a complex array from a single scalar for each real component and a single real array for the imaginary component.
+    /// C++ Interface to create a complex array from a single scalar for each
+    /// real component and a single real array for the imaginary component.
     ///
-    /// \param[in] real_ a single scalar to be assigned as the real component of each value of the returned complex array
-    /// \param[in] imag_ a real array to be assigned as the imaginary component of the returned complex array
-    /// \return the returned complex array
+    /// \param[in] real_ single scalar to be assigned as the real component of
+    ///                  each value of the returned complex array
+    /// \param[in] imag_ input array to be assigned as the imaginary component
+    ///                  of the returned complex array
+    /// \return          complex array
     AFAPI array complex(const double real_, const array &imag_);
     /// @}
 
     /// C++ Interface to return the real part of a complex array.
     ///
     /// \param[in] in input complex array
-    /// \return real part
+    /// \return       real part
```

Comment:
```suggestion
    /// \return    an array with the real values of the \p in array
```

Reply:
you need to look at the HTML to see that this is descriptive enough already.